dataplug/app/com/hubofallthings/dataplug/apiInterfaces/authProviders/DataPlugDisconnect.scala

```diff
@@ -13,6 +13,6 @@ import com.mohiva.play.silhouette.impl.providers.OAuth2Provider
 
 import scala.concurrent.Future
 
-trait HatOAuth2Provider extends OAuth2Provider {
+trait DataPlugDisconnect {
   def disconnect(phata: String, userId: String): Future[Done] = Future.successful(Done)
```

Comment:
one thing, there is no reason to tie this abstraction to the `akka` framework, so I would suggest a return type of `Future[Unit]`